test-project/Assets/Generated/Source/improbable/gdk/tests/ExhaustiveBlittableSingular.cs

```diff
@@ -367,6 +378,55 @@ public struct Snapshot : ISpatialComponentSnapshot
 
         public static class Serialization
         {
+            public static void SerializeComponent(Improbable.Gdk.Tests.ExhaustiveBlittableSingular.Component component, global::Improbable.Worker.Core.SchemaObject obj, global::Unity.Entities.World world)
+            {
+                {
```

Comment:
That's some weird-lookin' generated code...Why all the `{}`?

Reply:
To scope any local variables used in serialisation